apps/explorer-e2e/src/fixtures/constants.ts

```diff
@@ -0,0 +1,92 @@
+// General/Shared
+export const SEARCHBAR = `input[name="query"]`
+export const APP_NAME = 'explorer-identity-appName'
+export const APP_NETWORK = 'explorer-identity-network'
+export const DATUM_VALUE = 'explorer-datum-value'
+
+// Home page
+export const METRICS_ITEM = 'explorer-metrics-item'
+export const LATEST_BLOCKS_ITEM = 'explorer-latestBlocks-item'
+export const TOP_HOSTS_ITEMS = 'explorer-topHosts-item'
+
+// Contract page
+export const RENEWED_FROM_BUTTON = 'explorer-contract-renewedFrom'
+export const RENEWED_TO_BUTTON = 'explorer-contract-renewedTo'
+
+// Host page
+export const HOST_PRICING = 'explorer-hostPricing'
+export const HOST_SETTINGS_PATTERNS = [
+  '\\b\\d+(?:\\.\\d{1,2})?\\s?(TB|GB|MB|KB|B)\\b', // 3.00 TB
+  '(\\$\\d+(?:\\.\\d{1,3})?|\\d+(?:\\.\\d{1,3})?\\s?[A-Z]{2})\\/(TB|GB|MB|KB|B)\\/(month|year|day)', // $1.16/TB/month
+  '(\\$\\d+(?:\\.\\d{1,3})?|\\d+(?:\\.\\d{1,3})?\\s?[A-Z]{2})\\/(TB|GB|MB|KB|B)', // $1.12/TB
+  '\\b\\d+\\s?(days?|months?|years?)\\b', // 6 months
+  '^\\$\\d+(?:\\.\\d+)?$', // $4.50
+  '(\\$\\d+(?:\\.\\d+)?|\\d+(?:\\.\\d+)?\\s?[A-Z]{2})\\/million', // $0.0045 SC/million
+  '^\\d+(?:\\.\\d+)?\\s?(KS|mS)$', // 1.000 KS/mS
```

Comment:
I am not sure the value of these regex settings and price table tests as they will essentially match anything besides a string, it might provide a little more signal to grab a specific field and test against a specific one of these, but that also feels pretty redundant. Lets stick with this for now and see if we feel it provides some stability.